src/pages/blog/starlingx-deploying-your-app-3.md

```diff
@@ -0,0 +1,250 @@
+description: A very very very basic messaging exchange app
+appVersion: 1.5.2
+```
+
+and here's a look at the ```app_manifest.yaml``` after its configuration
+
+```sh
+(venv)~$: cat app_manifest.yaml
+
+---
+## App Manifest Configuration
+appManifestFile-config:
+  appName:  poc-starlingx-app
+  appVersion:  1.0.0
+  namespace:  default
+  chart:
+    - name:  poc-starlingx
+      appVersion:  1.5.2
+      path: ./helm-chart
+      chartGroup: poc-starlingx-chartGroup
+
+metadataFile-config:
+
+setupFile-config:
+  metadata:
+    author: Tomás Barros
+    url: https://dummy.com
+    classifier:
+      - "Operating System :: OS Independent"
+      - "License :: OSI Approved :: MIT License"
+      - "Programming Language :: Python :: 3"
+```
+
+Do you see how the helm-chart name and the appVersion in `Chart.yaml` matches the chart name in the `app_manifest.yaml`? This is no coincidence! They need to be matching exactly for the app-gen tool to work! If you want more details about this app-manifest configuration, please visit the app-gen-tool repository linked above.
+
+Now lets' run the app-gen-tool:
+
+```sh
+(venv)~$: stx-app-generator -i app_manifest.yaml -o ./output
+
+...
+Plugin wheels generated!
+Checksum generated!
+```
+
+There will be a bunch of information printed when this command is run, but if the last lines are as the ones above, the command was successful!
+
+Now, let's check what is inside the generated folder:
+
+```
+output
+└── poc-starlingx-app
+    ├── charts
+    │   └── poc-starlingx-1.5.2.tgz
+    ├── checksum.sha256
+    ├── fluxcd-manifests
+    │   ├── base
+    │   │   ├── helmrepository.yaml
+    │   │   ├── kustomization.yaml
+    │   │   └── namespace.yaml
+    │   ├── kustomization.yaml
+    │   └── poc-starlingx
+    │       ├── helmrelease.yaml
+    │       ├── kustomization.yaml
+    │       ├── poc-starlingx-static-overrides.yaml
+    │       └── poc-starlingx-system-overrides.yaml
+    ├── metadata.yaml
+    ├── plugins
+    │   ├── __init__.py
+    │   ├── k8sapp_poc_starlingx_app
+    │   │   ├── common
+    │   │   │   ├── constants.py
+    │   │   │   └── __init__.py
+    │   │   ├── helm
+    │   │   │   ├── __init__.py
+    │   │   │   └── poc_starlingx.py
+    │   │   ├── __init__.py
+    │   │   ├── kustomize
+    │   │   │   ├── __init__.py
+    │   │   │   └── kustomie_poc_starlingx_app.py
+    │   │   └── lifecycle
+    │   │       ├── __init__.py
+    │   │       └── lifecycle_poc_starlingx_app.py
+    │   ├── k8sapp_poc_starlingx_app-0.0.0-py2.py3-none-any.whl
+    │   ├── setup.cfg
+    │   └── setup.py
+    └── poc-starlingx-app-1.0.0.tgz
+``` 
+
+As you can see, the helm chart package was packed under the charts subdirectory. The next step would be to configure the plugins, since this is a complicated process and is application-dependent, we recommend you to read the README.MD of the app-gen-tool and reaching for help with the community in the StarlingX mail lists or in the matrix chat.
```

Comment:
Not only the plugins may need to be changed but also the static-overrides. Like you said it is something really complicated to be explained here, but also the README of the app-gen don't explain this very good, I think the best option is to link the wiki page, the amount of information there can be better suited for explaining to new users this specific process, although even there is not exactly greatly explained

Reply:
The truth is, there is nothing anywhere explaining how to do this, I guess is OK to leave the README, but the Wiki does have a lot of good information, I would link it too. Don't forget to add the information about the static-overrides, the text on the README is a good one.

Reply:
ok